articles/communication-services/concepts/call-automation/call-automation.md

```diff
@@ -160,10 +157,10 @@ To learn how to secure the callback event delivery, refer to [this guide](../../
 
 ## Known issues
```

Comment:
WOuld recommend pulling the latest changes - as this section has been removed. Similarly there might be more changes you are missing from the past week.